velox/exec/Task.cpp

```diff
@@ -168,9 +169,9 @@ Task::Task(
       queryCtx_(std::move(queryCtx)),
       pool_(
           queryCtx_->pool()->addChild(fmt::format("task.{}", taskId_.c_str()))),
-      splitPlanNodeIds_(collectSplitPlanNodeIds(planFragment_.planNode)),
       consumerSupplier_(std::move(consumerSupplier)),
       onError_(onError),
+      splitsStates_(initializeSplitStates(planFragment_.planNode)),
```

Comment:
```
If not a member function, then name it as buildSplitStates
```

Reply:
@xiaoxmeng 
I'l wait till tomorrow, maybe you'll come up with a new and better name. 😁